contrib/vexecutor/vagg.h

```diff
@@ -0,0 +1,94 @@
+#ifndef VAGG_H
+#define VAGG_H
+
+#include "executor/execHHashagg.h"
+
+/* batch hashagg group linklist header */
+typedef struct GroupData {
+	HashAggEntry *entry;	// pointer to agg_hash_entry
+	int idx; 				// pointer to idx_list
+} GroupData;
+
+/* batch hashagg group data */
+typedef struct BatchAggGroupData{
+	GroupData 	group_header[1024];	//group linklist header
```

Comment:
1024, use a macro is better.

Reply:
variable **BATCHSIZE** in *vexecutor.c* indicate the number of slots the TupleBatch hold.